### PR TITLE
added bottom nav bar , included binding

### PR DESCRIPTION
**Pull request by yellowHatpro**
- added bottom navigation bar along with fragments. No work in fragments though.
- included binding
- added fade animation on the transition between the two fragments

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:menu="@menu/bottom_menu"/>
+
+    <fragment
+        android:id="@+id/fragmentContainerView"
+        android:name="androidx.navigation.fragment.NavHostFragment"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
```
Comment (SunitRoy2703) [inline app/src/main/res/layout/activity_main.xml:24]: Please create a `dimens.xml`  and add the dimensions there

**Review by SunitRoy2703 — CHANGES_REQUESTED** [review]
Good work!
I've commented for some minor changes, let me when it's done.

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (yellowHatpro) [reply app/src/main/res/layout/activity_main.xml]: Thanks for pointing out sir,
```dimens.xml``` added
a few string values updates and stored in ```strings.xml```
replaced color code with @color/white

**Comment by SunitRoy2703** [comment]
Awesome! I'm merging it

**Pull request merged**